lib/chef/provisioning/docker_driver/driver.rb

```diff
@@ -105,7 +110,8 @@ def build_container(machine_spec, machine_options)
         target_repository = 'chef'
```

Comment:
But your example works with `docker_options[:from_image]` too. I think it is out of this PR scope.